README.md

```diff
@@ -0,0 +1,50 @@
+# Moneytree Rubocop Config
+
+Shared Rubocop config used by Moneytree's Ruby / Rails projects.
```

Comment:
If this is going to be open source, maybe a little more text here might be nice -- it's also an advertisement for new devs.  We might want to talk a little about how we value clean code, and we publish what we like in Ruby.

Reply:
I've added some words below this line. Could you take a look? (Especially for grammar, please)